### DISCUSSION TITLE
关于Alist的Crypt加密，未来如何自救？

### DISCUSSION THREAD

**Discussion by vicktw** [AlistGo/alist#7558] [Q&A]
我很喜欢使用Alist的Crypt加密上传文件到天翼云盘里面，但是我担心一个问题，假如某一天，天翼云盘做了一些技术性修改使得Alist无法继续挂载天翼云盘，又或者某一天，Alist崩了，无法继续使用？那么我要如何解密那些已经存放在天翼云盘里面的加密文件？
有无其他第三方解密软件可以自救？

**Comment by dgscyg** [comment]
> 我很喜欢使用Alist的Crypt加密上传文件到天翼云盘里面，但是我担心一个问题，假如某一天，天翼云盘做了一些技术性修改使得Alist无法继续挂载天翼云盘，又或者某一天，Alist崩了，无法继续使用？那么我要如何解密那些已经存放在天翼云盘里面的加密文件？ 有无其他第三方解密软件可以自救？

可以用 `Rclone` 来解密
https://alist.nn.ci/zh/guide/drivers/Crypt.html#%E9%AB%98%E7%BA%A7%E7%94%A8%E6%B3%95

**Comment by pongfcnkl** [comment]
把加密后文件存储路径改成本机存储路径就行了
另外那两个密码不要忘记，不然神仙都难救

**Comment by River-Codex** [comment]
> 我很喜欢使用Alist的Crypt加密上传文件到天翼云盘里面，但是我担心一个问题，假如某一天，天翼云盘做了一些技术性修改使得Alist无法继续挂载天翼云盘，又或者某一天，Alist崩了，无法继续使用？那么我要如何解密那些已经存放在天翼云盘里面的加密文件？ 有无其他第三方解密软件可以自救？

也不怕，实话， 网盘加密的数据还是可以直接下载到本地，再用旧版alist挂载crypt目录，再复制出来，很简单吧，是吧

**Reply by toutoumu** [reply]
刚刚测试了一下, 我手机里(Android,IOS)各装了一个AList, 两边的加密配置是一样的, 但是两边上传的文件无法相互访问, 是不是除了 密码和盐以外还有其他的什么控制了, 我从导出的配置里看到的加密后的密码和盐是不一样的. 即便我将对方(IOS,Android)的配置文件导入也无法访问.

**Reply by pongfcnkl** [reply]
> 刚刚测试了一下, 我手机里(Android,IOS)各装了一个AList, 两边的加密配置是一样的, 但是两边上传的文件无法相互访问, 是不是除了 密码和盐以外还有其他的什么控制了, 我从导出的配置里看到的加密后的密码和盐是不一样的. 即便我将对方(IOS,Android)的配置文件导入也无法访问.

加密后文件存储路径呢？
